src/multi_vector_simulator/E0_evaluation.py

```diff
@@ -186,6 +187,11 @@ def evaluate_dict(dict_values, results_main, results_meta):
     E4.detect_excessive_excess_generation_in_bus(dict_values)
     E4.verify_state_of_charge(dict_values)
 
+    # Format KPI about energy generation into a pd.DataFrame
+    dict_values[KPI][KPI_UNCOUPLED_DICT] = pd.DataFrame.from_dict(
+        dict_values[KPI][KPI_UNCOUPLED_DICT], orient="index"
+    )
+
```

Comment:
Is this a workaround for the EPA, or does this mean that we will keep adding parameters to the `KPI_UNCOUPLED_DICT` as we were, and soley reformat it so that it is easier to use in the report? 

Does this fix how the values are printed in the excel-sheet?

Reply:
It is no workaround, it makes a pd.DF from the way we were adding the parameters (it does it at the very end of the E0 function. It does print it nicely, in the report, and in the xls sheet. The EPA parser will be done in #675 after rebasing on dev after merging this PR :)